datastore/adapter/mount.py

```diff
@@ -12,28 +12,25 @@
 ]
 
 
+mount_item_t = typing.Tuple['mount_tree_t[DS]', typing.Optional[DS]]
+
+
+class mount_tree_t(typing.Dict[str, mount_item_t[DS]]):
+	...
+
```

Comment:
Yeah, that looks like a better solution. :+1:

Reply:
This change was made by you, not me. I hope you aren't going through all commits. Just go through the last 1-2 commits (the ones made by me)